Cleanup changelog

deivid-rodriguez opened the pull request.

varyonic commented:
I guess this works fine on GitHub but maybe not elsewhere? http://www.cirosantilli.com/markdown-style-guide/#reference-style-links

deivid-rodriguez commented:
Thanks @varyonic, nice resource! I removed the commit and added a few other markdown changes that were not limited to the change log. So I'm closing this PR and will follow up in #5278.
